schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id GENERATED ALWAYS AS IDENTITY, 
```

Comment:
Optional 
  you can also mark the id column as a primary key 😄 
`PRIMARY KEY (id)`